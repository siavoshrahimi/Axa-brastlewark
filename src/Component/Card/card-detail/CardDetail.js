import React from 'react';

import "./cardDetail.scss"

const CardDetail = ({name,age}) => {
    return(
        <div className='card-detail'>
            <span>Name:{name}</span>
            <span>Age:{age}</span>
        </div>
    )
}

export default CardDetail;