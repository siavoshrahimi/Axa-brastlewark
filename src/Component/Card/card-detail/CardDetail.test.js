import React from 'react';
import {shallow} from 'enzyme';

import CardDetail from './CardDetail';


it('expect to render card detail component' ,() =>{
    expect(shallow(<CardDetail />)).toMatchSnapshot();
})