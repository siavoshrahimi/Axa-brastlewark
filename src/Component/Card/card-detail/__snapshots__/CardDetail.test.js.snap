// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render card detail component 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CardDetail />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <span>
          Name:
        </span>,
        <span>
          Age:
        </span>,
      ],
      "className": "card-detail",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Name:",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          "Name:",
          undefined,
        ],
        "type": "span",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "Age:",
            undefined,
          ],
        },
        "ref": null,
        "rendered": Array [
          "Age:",
          undefined,
        ],
        "type": "span",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <span>
            Name:
          </span>,
          <span>
            Age:
          </span>,
        ],
        "className": "card-detail",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Name:",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Name:",
            undefined,
          ],
          "type": "span",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "Age:",
              undefined,
            ],
          },
          "ref": null,
          "rendered": Array [
            "Age:",
            undefined,
          ],
          "type": "span",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
